src/main/java/com/dimsirka/animalservice/controllers/AnimalController.java

```diff
@@ -77,6 +78,11 @@ public String getAll(@RequestParam(name = "page", required = false, defaultValue
         return "animal/list";
     }
 
+    @GetMapping({"/about"})
+    public String getAll() {
+        return "animal/about";
+    }
+
     @GetMapping("/animals")
```

Comment:
Please, create a separate controller HomeController and place this endpoint here.

Reply:
Created Separated Controller